chart/values.yaml

```diff
@@ -36,7 +36,8 @@ debug: false
 
 # -v, -vv, -vvv
 logLevel: ""
-jsonLogs: true
+# This has to be stringy because of the way templates get evaluated
+jsonLogs: "true"
```

Comment:
I am not sure we want to expose this "hack"  - why not just use global.jsonLogs ?

Reply:
Using `tpl` is the only way to template values from the same file

If in the mission-control chart, we want to `global.jsonLogs` to be passed here, we have to use `tpl` but that does not work with bools

Reply:
We can have something like this: https://github.com/Masterminds/sprig/issues/258

but thats even more hacky